Makefile

```diff
@@ -38,17 +26,21 @@ help:
 	@echo
 
 .PHONY: build
-build: docker-build-db .build/requirements.timestamp compile-catalog c2cgeoform/static/dist/index.js
+build: docker-build-db poetry compile-catalog c2cgeoform/static/dist/index.js
 
 .PHONY: check
-check: flake8 check_c2cgeoform_demo
+check: prospector check_c2cgeoform_demo
+
+.PHONY: poetry
+poetry:
+	poetry install --with=dev --extras=psycopg2-binary
 
-.PHONY: flake8
-flake8: .build/requirements-dev.timestamp
-	$(VENV_BIN)/flake8 --exclude=node_modules c2cgeoform
+.PHONY: prospector
+prospector: poetry
```

Comment:
Why not running prospector using pre-commit ?

Reply:
For tow reason:
- Proespector run is too slow: pre-comit run in less than 10s. (8.6s. on my host on all files, on second run) and prospector in 1m45s (same condition).
- Prospector needs the project dependencies